### PR TITLE
restructured publishing options

### PR DESCRIPTION
**Pull request by paolap**
Hi, this is my attempt to restructured the publish options section as we discussed.
I also added content to some of the publishing-options file itself. It's a really rough draft and mostly is copied and pasted from the CMS wiki, so I still need t do a lot of work before is worth to review it.
I created a pull request mostly to make sure the new structure is shared earlier than later.
And I left the clex bit in, so to give an idea of what the other parts of that page (i.e. CSiro and BoM) might look like. Said so, these will probably have less options.
The details of the procedures should be in the other sections (for ex. publish-csiro-dap.md), so feel free to had a new file if there's a specific BoM procedure you want to list too.

### CODE REVIEW

File: Governance/publish/publish-csiro-dap.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
...
+# CSIRO Data Access Portal (DAP) 
 
 Data, software and links to external data holdings (such as NCI) can be added to the [CSIRO Data Access Portal](https://data.csiro.au/) (DAP) by **CSIRO staff** only. More information on creating metadata records and uploading files to the CSIRO DAP can be found on the [DAP Help Guide](https://confluence.csiro.au/display/dap/Deposit+and+Manage+Data) (staff only access).
 
  CSIRO-affiliated data can be published in the DAP and the lead creator does not have to be CSIRO staff member.
+
+!!!COMMENT: at the moment this is identical to what we have in the Publishing option pages for CSIRO. I'm leaving this here for the moment in case we want to add something more, like:
+ - summary of the procedure with pros and cons? 
+ - Something which is not covered in the official documentation but could be useful?  
+ - a reminder to follow discipline specific best practices even if they aren't encessarily required?
```
Comment (hot007) [inline Governance/publish/publish-csiro-dap.md:10]: Happy to approve this for now and Katie and I can expand later.

File: Governance/publish/publish-esgf.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# ESGF publishing procedure
```
Comment (hot007) [inline Governance/publish/publish-esgf.md:1]: @chloemackallah to populate these once merged I assume?

**Review by hot007 — APPROVED** [review]
Happy to approve this PR, once merged we can build on the current skeleton parts.

File: Governance/publish/publish-esgf.md (hunk 2 above)
Comment (paolap) [reply Governance/publish/publish-esgf.md:1]: Yes and Avi can work on the BoM part I'm waiting on the merge before adding him.

**Comment by paolap** [comment]
Just so you know I've fixed some typos and tidied up the CLEX content part of the publishing-options.

**Review by chloemackallah — APPROVED** [review]

**Pull request merged**